src/components/detail/detail.style.ts

```diff
@@ -46,7 +49,7 @@ const StyledDetailIcon = styled(Icon)`
   }
 `;
 
-const StyledDetailFootnote = styled.div`
+const StyledDetailFootnote = styled.div<IconProps>`
```

Comment:
Similar to previous comment :)
```suggestion
const StyledDetailFootnote = styled.div<{ hasIcon: boolean }>`
```